daisy_workflows/image_build/windows_for_containers/container_image_install.ps1

```diff
@@ -189,12 +310,17 @@ function Run-SecondBootSteps {
 & ping 127.0.0.1 -n 60
 try {
   $windows_version = Get-MetadataValue 'version'
+
+  # Use the existence of the docker service to determine if all firstboot
+  # steps completed, as the container install forces a reboot prior to
+  # docker installation.
+  $dockerStatus = Get-Service -Name 'docker' -ErrorAction SilentlyContinue
   Write-Host "Windows version: $windows_version"
   if (-not $windows_version) {
     throw 'Error retrieving "version" from metadata'
   }
 
-  if (!(Get-Package -ProviderName DockerMsftProvider -ErrorAction SilentlyContinue| Where-Object {$_.Name -eq 'docker'})) {
+  if ($dockerStatus -eq $null) {
     Run-FirstBootSteps
```

Comment:
I'm already aware this is a misnomer since this will technically run twice per build, but we need to get this operational asap. Renaming/breaking out this block can be something we can do later, since this approach will successfully install and configure the docker runtime.